src/review.tsx

```diff
@@ -38,89 +37,89 @@ const ReviewForm = () => {
         let prediction = JSON.parse(XHR.responseText).prediction;
         if (prediction == 0) {
           setSentiment(false);
-        } 
+        }
         // We received a sentiment without errors so we change the state
-        setSentimentReceived(true)
-      } 
+        setSentimentReceived(true);
+      }
       // If we get a server of client error show error message
       if (/(4|5)\d\d/.test(String(XHR.status))) {
-        setErrorOccurred(true)
+        setErrorOccurred(true);
       }
     };
-  
+
     // Set up our request
-    XHR.open("POST", process.env.MODEL_SERVICE_URL+"predict");
-  
+    XHR.open("POST", process.env.MODEL_SERVICE_URL + "predict");
```

Comment:
we should align on conventions with slashes being present for routes or not. Let's put slashes here, rather than in the environment variable as I would also need to update it for the operation repo